js/service/BoardService.js

```diff
@@ -26,32 +26,82 @@ app.factory('BoardService', function(ApiService, $http, $q){
     };
     BoardService.prototype = angular.copy(ApiService.prototype);
 
-    BoardService.prototype.searchUsers = function(search) {
-        var url = OC.generateUrl('/apps/deck/share/search/'+search);
```

Comment:
I'm not familiar enough with the app, but if you're not calling this endpoint any more, shouldn't it be removed?